cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ProcessConfiguration.java

```diff
@@ -17,11 +18,18 @@
     protected int maxHeapMemorySize = -1;
 
     protected Collection<String> jvmOptions;
+    protected Collection<String> processParameters = new ArrayList<>();
+
 
     public ProcessConfiguration(ServiceEnvironmentType environment, int maxHeapMemorySize, Collection<String> jvmOptions) {
+        this(environment, maxHeapMemorySize, jvmOptions, new ArrayList<>());
+    }
+
+    public ProcessConfiguration(ServiceEnvironmentType environment, int maxHeapMemorySize, Collection<String> jvmOptions, Collection<String> processParameters) {
         this.environment = environment;
         this.maxHeapMemorySize = maxHeapMemorySize;
         this.jvmOptions = jvmOptions;
+        this.processParameters = processParameters == null ? new ArrayList<>() : processParameters;
```

Comment:
And why is the null check there?

Reply:
Had a null pointer there with an old task configuration.
I will check if only this is relevant(or if the review you suggested before is also relevant)

Reply:
Haven't tested it yet but I don't think it would work without.
I could do it like this if you want: it doesn't create a new ArrayList when the parameter is null and just use the initialized one

Reply:
You have initialize the field direcly or in the getter

Reply:
I am allready initializing it directly. But I think null is getting parsed as a constructor parameter. I will make an if statement there to check if the constructor parameter is not null